app/javascript/packages/document-capture/components/document-capture-not-ready.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button } from '@18f/identity-components';
+import { useI18n } from '@18f/identity-react-i18n';
+import { useContext } from 'react';
+import FlowContext from '@18f/identity-verify-flow/context/flow-context';
+import { addSearchParams, forceRedirect, Navigate } from '@18f/identity-url';
+import { getConfigValue } from '@18f/identity-config';
+import AnalyticsContext from '../context/analytics';
+import { ServiceProviderContext } from '../context';
+
+export interface DocumentCaptureNotReadyProps {
+  navigate?: Navigate;
+}
+
+function DocumentCaptureNotReady({ navigate }: DocumentCaptureNotReadyProps) {
+  const { t } = useI18n();
+  const { trackEvent } = useContext(AnalyticsContext);
+  const { currentStep } = useContext(FlowContext);
+  const { name: spName, failureToProofURL } = useContext(ServiceProviderContext);
+  const appName = getConfigValue('appName');
+  const header = <h2 className="h3">{t('doc_auth.not_ready.header')}</h2>;
+
+  const content = (
+    <p>
+      {spName
+        ? t('doc_auth.not_ready.content_sp', {
+            sp_name: spName,
+            app_name: appName,
+          })
+        : t('doc_auth.not_ready.content_nosp', {
+            app_name: appName,
+          })}
+    </p>
+  );
+  const handleExit = () => {
+    trackEvent('IdV: docauth not ready link clicked');
+    forceRedirect(
+      addSearchParams(spName ? failureToProofURL : '/account', {
+        step: currentStep,
+        location: 'not_ready',
+      }),
+      navigate,
+    );
+  };
+
+  return (
+    <>
+      {header}
+      {content}
```

Comment:
This works, but there are two ways to make it more in line with our FE patterns:

One, put everything inline. These are small enough that this is probably the easiest choice:
```javascript
<>
      <h2 className="h3">{t('doc_auth.not_ready.header')}</h2>
      <p>
        {spName
          ? t('doc_auth.not_ready.content_sp', {
              sp_name: spName,
              app_name: appName,
            })
          : t('doc_auth.not_ready.content_nosp', {
              app_name: appName,
            })}
      </p>
...etc...
</>
```

Two make these into components. This is the "Correct according to React" way and probably the most reusable way to do this:
```javascript
// These go outside the `DocumentCaptureNotReady` component
const NotReadyHeader = ( {text} ) => <h2 className="h3">{ text }</h2>;
const NotReadyContent = ( {text} ) => <p> { text } </p>

// This goes in the `DocumentCaptureNotReady` component
const headerText = t('doc_auth.not_ready.header')
const contentText = spName
        ? t('doc_auth.not_ready.content_sp', {
            sp_name: spName,
            app_name: appName,
          })
        : t('doc_auth.not_ready.content_nosp', {
            app_name: appName,
          })
// This is in the main return
<>
      <NotReadyHeader text={headerText} />
      <NotReadyContent text={contentText} />
...etc...
</>

```

Reply:
@charleyf  I usually do SP testing from local by http://localhost:3000/test/saml/login, this only SAML though but it does provide SP information.

Reply:
@charleyf, indeed they are small, I put them inline.

Reply:
> @charleyf I usually do SP testing from local by http://localhost:3000/test/saml/login, this only SAML though but it does provide SP information.

Whoa, TIL this exists!

On a related note, I proposed some documentation to refer to sample apps in #9558, but that test controller could save a few steps as long as you can remember/bookmark the URL.

Reply:
@aduth, it is in the rails route. 😄

Reply:
Should we also have a /test/oidc/login route?

Reply:
It could probably make sense to have one if we think it'd be useful, even just to maintain balance between the two protocols 😄